02-Advanced-topics/05-GSD-visualization.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>This section looks like it belongs at the end of the box tutorial. Or it just isn't needed at all because "we do not recommend" it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/fresnel-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
It felt too "advanced" for the basic tutorial and since its not something recommended, I will just drop it.